ptx/sec_differentials.ptx

```diff
@@ -408,15 +410,26 @@
     studied in introductory Differential Equations courses.
   </p>
 
-  <aside vshift="0">
-    <title>PID controllers</title>
+  <aside vshift="3">
+    <title>Applications of Differentials</title>
     <p>
-      Another place differentials are used is in a PID controller,
+      Differentials are used within <q>Proportional Integral Derivative</q> (PID) controllers which use both integral and differential calculus to accurately control a process.
+		</p>
+		<p>
+			For instance, consider the task of steering a self--driving car. If the vehicle drifts to the left, how much should the wheel be turned to the right to correct the path? Too little and the problem may not be corrected at all; too much and the car will overshoot its target and extreme corrective action will need to be taken. PID controllers consider the <em>rate</em> at which the drift and corrective measures take effect.
+		</p>
```

Comment:
The `--` for a hyphen is LaTeX code, but it is not valid PreTeXt. You want to use <ndash/>.

Reply:
Use <ndash/>?

Reply:
Ha! Github does like writing the pretext code. Use < n d a s h / >?  (Without spaces - Github turns that command into a space.)

Reply:
You can put pretext code between back ticks and it works OK. Which I clearly forgot to do!

So yes, you would do `self<ndash/>driving`.